numpy/core/numeric.py

```diff
@@ -971,21 +971,43 @@ def tensordot(a, b, axes=2):
         * ``axes = 1`` : tensor dot product :math:`a\\cdot b`
         * ``axes = 2`` : (default) tensor double contraction :math:`a:b`
 
-    When `axes` is integer_like, the sequence for evaluation will be: first
-    the -Nth axis in `a` and 0th axis in `b`, and the -1th axis in `a` and
-    Nth axis in `b` last.
+    When `axes` is integer_like, the sequence of axes for evaluation
+    will be: from the -Nth axis to the -1th axis in `a`, 
+    and from the 0th axis to (N-1)th axis in `b`.
+    For example, ``axes = 2`` is the equal to
+    ``axes = [[-2, -1], [0, 1]]``.
+    When N-1 is smaller than 0, or when -N is larger than -1,
+    the element of `a` and `b` are defined as the `axes`.
 
     When there is more than one axis to sum over - and they are not the last
     (first) axes of `a` (`b`) - the argument `axes` should consist of
     two sequences of the same length, with the first axis to sum over given
     first in both sequences, the second axis second, and so forth.
+    The calculation can be referred to ``numpy.einsum``.
 
     The shape of the result consists of the non-contracted axes of the
     first tensor, followed by the non-contracted axes of the second.
 
     Examples
-    --------
-    A "traditional" example:
+    -------- 
+    An example on integer_like
+
+    >>> a_0 = np.array([[1, 2], [3, 4]])
+    >>> b_0 = np.array([[5, 6], [7, 8]])
+    >>> c_0 = np.tensordot(a_0, b_0, axes=0)
+    >>> c_0.shape
+    (2, 2, 2, 2)
+    >>> c_0
+    array([[[[ 5,  6],
+             [ 7,  8]],
+            [[10, 12],
+             [14, 16]]],
+           [[[15, 18],
+             [21, 24]],
+            [[20, 24],
+             [28, 32]]]])
+
```

Comment:
I believe there should be blank lines in between see: 
```
>>> import numpy as np
>>> a_0 = np.array([[1, 2], [3, 4]])
>>> b_0 = np.array([[5, 6], [7, 8]])
>>> c_0 = np.tensordot(a_0, b_0, axes=0)
>>> c_0.shape
(2, 2, 2, 2)
>>> c_0
array([[[[ 5,  6],
         [ 7,  8]],

        [[10, 12],
         [14, 16]]],


       [[[15, 18],
         [21, 24]],

        [[20, 24],
         [28, 32]]]])
```

Reply:
Thanks, linus-md, but I think there should not be some blank lines. Because I checked other codes and found they didn't include blank lines.

Reply:
No you need to include the actual output as far as I know. Did you run your example?

Reply:
You only had one empty line after the second block. If this it is not the problem I don't know what else it could be.

Reply:
yes, I run the example, I deleted the empty lines because other examples did that
![image](https://github.com/user-attachments/assets/52c9cde8-f0e9-4377-8f10-18c7db2ba4a9)
I think it make sense, because blank line may split the output into two parts.

I think the codes is fine now, I fixed the conflicts that is caused by the location move.